pkg/sessions/redis/redis_store.go

```diff
@@ -290,7 +293,7 @@ func newTicket() (*TicketData, error) {
 		return nil, fmt.Errorf("failed to create new ticket ID %s", err)
 	}
 	// ticketID is hex encoded
-	ticketID := fmt.Sprintf("%x", rawID)
+	ticketID := hex.EncodeToString(rawID)
```

Comment:
Does this change the output at all? Just concerned about existing tickets

Reply:
I think this change will return exactly the same result.
If you do not like it, I will back to the original implementation.
https://play.golang.org/p/ijrr87i_rox